explainaboard/convert.py

```diff
@@ -0,0 +1,42 @@
+import csv
```

Comment:
This seems like a script that shouldn't be checked in?

Reply:
Thanks! And yeah, I didn't notice I checked in that script too. I have removed it in a new commit.